apps/examples/binary_update/binary_update_main.c

```diff
@@ -84,7 +84,7 @@ static void binary_update_getinfo(char *name)
 	binary_info_t bin_info;
 
 	printf("\n** Binary Update GETINFO [%s] test.\n", name);
-	ret = binary_update_get_binary_info(name, &bin_info);
+	ret = binary_manager_update_get_binary_info(name, &bin_info);
```

Comment:
Basic naming is do-What.
In your case, do - done - what
so, binary_manager(prefix)_update(do)_binary_info(what) is enough.

Reply:
Actually, bin_info got from this function is data for binary update.
So I made a name, binary_manager_update(prefix)_get(do)_binary_info(what).
I will consider this name more.